tee-worker/app-libs/stf/src/trusted_call_litentry.rs

```diff
@@ -133,11 +152,11 @@ impl TrustedCallSigned {
 			StfError::RequestVCFailed(assertion, ErrorDetail::UserShieldingKeyNotFound)
 		);
 
-		let id_graph = IMT::get_id_graph(&who, usize::MAX);
+		let id_graph = IMT::get_id_graph_with_only_active_identities(&who, usize::MAX);
```

Comment:
I think this one overlaps with the `filter` below, doesn't it

maybe we don't need `get_id_graph_with_only_active_identities` at all - it's only used when building assertions anyway

Reply:
Yes it does, thanks.